cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperator.java

```diff
@@ -156,7 +156,11 @@ public void eventReceived(Action action, ConfigMap cm) {
                                     if (result.succeeded()) {
                                         log.info("{}: assembly reconciled", reconciliation);
                                     } else {
-                                        log.error("{}: Failed to reconcile", reconciliation, result.cause());
+                                        if (result.cause() instanceof InvalidConfigMapException) {
+                                            log.error("{}: Failed to reconcile", reconciliation, result.cause().getMessage());
```

Comment:
warn level is sufficent and I think you're missing a `{}` for the exception message.

Reply:
TBH I would factor out a local var for `result.cause()`, since you're calling it 3 times here.